src/record/internal/au3/au3record.h

```diff
@@ -4,11 +4,13 @@
 
 #pragma once
 
+#include "actions/iactionsdispatcher.h"
 #include "global/async/asyncable.h"
 
-#include "modularity/ioc.h"
-#include "context/iglobalcontext.h"
 #include "au3audio/iaudioengine.h"
+#include "context/iglobalcontext.h"
+#include "irecordcontroller.h"
+#include "modularity/ioc.h"
```

Comment:
please move it back to the top of the group
the order should be as follows: 
ioc - the main include without which other includes will not work
includes from the framework
includes from the other audacity modules
includes from the current module